protractor_spec.js

```diff
@@ -18,7 +18,8 @@ var IGNORED_TESTS = [
   'closure/goog/net/jsloader_test.html',
   'closure/goog/net/filedownloader_test.html',
   'closure/goog/promise/promise_test.html',
-  'closure/goog/editor/plugins/abstractdialogplugin_test.html'
+  'closure/goog/editor/plugins/abstractdialogplugin_test.html',
```

Comment:
I wonder if `closure/goog/editor/plugins/abstractdialogplugin_test.html` is fixed by the following commit:
https://github.com/google/closure-library/commit/a1b20191b1cfd7937b1a3402eb1347bd54bc3116#diff-c0a7f4c78a87dc0d9c753419bf2fd040R89